code/modules/mob/living/silicon/robot/component.dm

```diff
@@ -183,12 +211,17 @@
 // Initializes cyborg's components. Technically, adds default set of components to new borgs
 /mob/living/silicon/robot/proc/initialize_components()
 	components["actuator"] = new/datum/robot_component/actuator(src)
+	actuatorComponent = components["actuator"]
 	components["radio"] = new/datum/robot_component/radio(src)
 	components["power cell"] = new/datum/robot_component/cell(src)
 	components["diagnosis unit"] = new/datum/robot_component/diagnosis_unit(src)
 	components["camera"] = new/datum/robot_component/camera(src)
 	components["comms"] = new/datum/robot_component/binary_communication(src)
 	components["armour"] = new/datum/robot_component/armour(src)
+	components["jetpack"] = new/datum/robot_component/jetpack(src)
+	jetpackComponent = components["jetpack"]
+	var/datum/robot_component/JC = components["jetpack"]
+	JC.installed = 0//We start the jetpack as not installed, because its nondefault

```

Comment:
Why is this necessary, I'm curious? It technically installs it, but then changes a variable to make it "appear" not installed. While still being there?

Why not just leave it uninstalled?